lib/src/builders/application.dart

```diff
@@ -43,8 +46,16 @@ class ApplicationUpdateBuilder extends UpdateBuilder<Application> {
         if (installationParameters != null)
           'install_params': {
             'scopes': installationParameters!.scopes,
-            'permissions': installationParameters!.permissions.toString(),
+            'permissions': installationParameters!.permissions.value.toString(),
           },
+        if (integrationTypesConfig != null)
+          'integration_types_config': integrationTypesConfig!.map((key, value) => MapEntry(key.value.toString(), {
+                if (value.oauth2InstallParameters != null)
+                  'oauth2_install_params': {
+                    'scopes': value.oauth2InstallParameters!.scopes,
+                    'permissions': value.oauth2InstallParameters!.permissions.value.toString(),
+                  },
+              })),
```

Comment:
```suggestion
        if (integrationTypesConfig != null)
          'integration_types_config': {
            for (final MapEntry(:key, :value) in integrationTypesConfig!.entries)
              key.value.toString(): {
                if (value.oauth2InstallParameters != null)
                    'oauth2_install_params': {
                      'scopes': value.oauth2InstallParameters!.scopes,
                      'permissions': value.oauth2InstallParameters!.permissions.value.toString(),
                    },
              },
          }
```

Also, could we have an `ApplicationIntegrationTypeConfigurationBuilder` instead of reusing the `ApplicationIntegrationTypeConfiguration` type?